drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -119,6 +129,9 @@ def createNewImagingRun(tr, base_sample, exp, omero_image_ids, run_offset, prope
 def getFileFromLine(line):
 	return line.split("\t")[0]
 
+def getImageFolderPathFromLine(line):
+	return line.split("\t")[0]
```

Comment:
These methods do the same thing, I'd recommend to diversify them by e.g. checking if the line is a valid path for a folder or file?

Reply:
Good point, `getFileFromLine()` was redundant, I removed it